lms/djangoapps/badges/utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Utility functions used by the badging app.
+"""
+from django.conf import settings
+
+
+def site_prefix():
+    """
+    Get the prefix for the site URL-- protocol and server name.
+    """
+    scheme = u"https" if settings.HTTPS == "on" else u"http"
+    return u'{}://{}'.format(scheme, settings.SITE_NAME)
```

Comment:
Can't we get that from the request object, or simply use protocol-relative URLs?  Or do we need that while running management commands?